dungeon/src/contrib/entities/MiscFactory.java

```diff
@@ -26,28 +26,40 @@
 public final class MiscFactory {
 
   private static final Random RANDOM = new Random();
+  private static final int DEFAULT_CHEST_SIZE = 12;
+  private static final int MAX_AMOUNT_OF_ITEMS_ON_RANDOM = 5;
+  private static final int MIN_AMOUNT_OF_ITEMS_ON_RANDOM = 1;
 
   /**
-   * Get an Entity that can be used as a chest.
-   *
-   * <p>Will contain some random items.
+   * This method is used to create a new chest entity. The chest entity can either be empty or
+   * filled with random items.
    *
-   * <p>The Entity is not added to the game yet. *
+   * <p>The Entity is not added to the game yet.
    *
-   * <p>It will have a {@link InteractionComponent}. {@link PositionComponent}, {@link
+   * <p>The Entity will have a {@link InteractionComponent}, {@link PositionComponent}, {@link
    * core.components.DrawComponent}, {@link contrib.components.CollideComponent} and {@link
-   * contrib.components.InventoryComponent}. It will use the * {@link
+   * contrib.components.InventoryComponent}. It will use the {@link
    * contrib.utils.components.interaction.DropItemsInteraction} on interaction.
    *
-   * @return A new Entity.
-   * @throws IOException if the animation could not been loaded.
+   * @param type The type of chest to be created. It can either be RANDOM (filled with random items)
+   *     or EMPTY.
+   * @return A new Entity representing the chest.
+   * @throws IOException if the animation could not be loaded.
    */
-  public static Entity newChest() throws IOException {
-    Set<Item> items =
-        IntStream.range(0, RANDOM.nextInt(1, 3))
-            .mapToObj(i -> ItemGenerator.generateItemData())
-            .collect(Collectors.toSet());
-    return newChest(items, PositionComponent.ILLEGAL_POSITION);
```

Comment:
@Flamtky Solche Änderungen sind nicht unbedingt rückwärtskompatibel - ich habe keine richtige Idee, wer das Ding alles nutzt. 

Ich würde in diesem Fall die alte Methode drin lassen und mit Deiner neuen Methode überladen. Dabei könnte es dann eine Weiterleitung in der alten (parameterfreien) Methode auf die neue Methode geben. Was denkst Du?

Reply:
On 2nd thought: Mein Einwand kann auch komplett Banane sein. Wenn ich Deinen neuen Code sehe, müsste man die alte Methode vermutlich auch entsprechend anpassen, oder? 

Du hast sicher mal gesucht, ob und wo die alte Methode vorkam. Wenn Du denkst, wir brauchen das alte Ding nicht mehr, dann kannst Du einfach den Kommentar auf "resolved" stellen und mergen ...

Reply:
Habe einfach wie Methode wieder hinzugefügt. Das Ergebnis war ja einfach eine Kiste mit zufälligen Items